packages/graphile-build-pg/src/plugins/PgColumnDeprecationPlugin.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import type { Plugin } from "graphile-build";
+
+export default (function PgColumnDeprecationPlugin(builder) {
+  builder.hook("GraphQLObjectType:fields:field", (field, build, context) => {
+    const {
+      scope: {
+        isPgRowType,
+        isPgCompoundType,
+        pgIntrospection: table,
+        pgFieldIntrospection,
+      },
+    } = context;
+    if (
+      !(isPgRowType || isPgCompoundType) ||
+      !table ||
+      table.kind !== "class" ||
+      !pgFieldIntrospection ||
+      !pgFieldIntrospection.tags
+    ) {
+      return field;
+    }
+    return pgFieldIntrospection.tags.deprecated
+      ? Object.assign({}, field, {
+          deprecationReason: pgFieldIntrospection.tags.deprecated,
```

Comment:
Feel free to follow up with a PR that changes this to:

Reply:
```
          deprecationReason: Array.isArray(pgFieldIntrospection.tags.deprecated) ? pgFieldIntrospection.tags.deprecated.join("\n") : pgFieldIntrospection.tags.deprecated,
```

(I can't do via GitHub because of prettier and don't have time to check it out.)